Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -7405,6 +7405,12 @@
             "state" : "translated",
             "value" : "Find people"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : ""
```

Comment:
This doesn't seem right. I will try to reject it in Crowdin.

Reply:
I [did](https://crowdin.com/editor/nossocial/420/en-es?view=comfortable&filter=basic&value=0#q=Find%20people). Not sure how long it needs to update the PR.

Reply:
I don't like the other change, either, so I'm going to suggest we just close this PR.